core/src/services/webdav/fixtures/docker-compose-webdav-with-empty-passwd.yml

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '3.8'
+
+services:
+  webdav:
+    image: dgraziotin/nginx-webdav-nononsense
+    ports:
+      - "8080:80"
+    volumes:
+      - ./data:/data
```

Comment:
```suggestion
```

We don't need to mount `./data` into container (since we don't need to read it).

Reply:
Removed

Reply:
Seems removing will cause docker have problems.

Reply:
We can use `webdav-data:/data` instead.